src/Domain/User/UserKeyRotator.ts

```diff
@@ -0,0 +1,26 @@
+import { inject, injectable } from 'inversify'
+import * as cryptoRandomString from 'crypto-random-string'
```

Comment:
Why are we using a 3rd party package for this? This is dangerous for crypto-related utility functions, especially when Node makes this really easy. We should create our own utility function for this. Will probably be 3 lines.